front/src/app/components/forms/convention/conventionHelpers.ts

```diff
@@ -33,11 +33,28 @@ type WithFromPeConnectedUser = {
 };
 
 export type ConventionPresentation = OmitFromExistingKeys<
-  Partial<ConventionDto>,
-  "statusJustification"
+  Partial<ConventionReadDto>,
+  | "agencyName"
+  | "agencyCounsellorEmails"
+  | "agencyValidatorEmails"
+  | "agencySiret"
 > &
   WithSignatures &
   WithEstablishmentTutor &
   WithIntershipKind &
-  WithAgencyDepartment &
   WithFromPeConnectedUser;
+
+export const conventionPresentationSchema: z.Schema<ConventionPresentation> =
+  conventionSchema.and(
+    z.object({
+      agencyDepartment: z.string(),
+      agencyKind: agencyKindSchema,
```

Comment:
Est-ce que ça suffirait pas ça (pour fixer ce qu'on a vu tout à l'heure) :

```suggestion
      agencyKind: agencyKindSchema.optional(),
```

Reply:
nope ça suffit pas